spec/services/admin/participants/search_spec.rb

```diff
@@ -141,31 +139,6 @@
           expect(results).not_to include(pp_2, pp_3)
         end
       end
-
-      describe "matching by teacher reference number" do
-        let(:search_term) { user_1.npq_applications.first.teacher_reference_number }
-
-        let(:results) { search.call(ParticipantProfile, search_term:) }
-
-        it "returns matching participants" do
-          expect(results).to include(pp_1)
-        end
-
-        it "doesn't return non-matching participants" do
-          expect(results).not_to include(pp_2, pp_3)
-        end
-      end
```

Comment:
I think we should keep this test, but change it for `.or(TeacherProfile.trn_matches(search_term))`

Also on `Admin::Participants::Search `, we can remove:
```ruby
        .or(NPQApplication.where(teacher_reference_number: search_term))
        .or(NPQApplication.where(id: search_term))
```

Reply:
Hi @mooktakim Addressed in [3a38988](https://github.com/DFE-Digital/early-careers-framework/pull/5358/commits/3a3898865a874725f0babbc4cfc88d72e9b9d2b2). Tksss. Also I noticed there's already another spec in the same file testing the search by TRN..